include/utils/types.hpp

```diff
@@ -96,6 +76,30 @@ struct Mapping {
     }
 };
 
+/// \brief Writes the string representation of the given Mapping \p mapping to the given output stream \p os
+/// \returns an output stream with the Mapping written to
+inline std::ostream& operator<<(std::ostream& os, const Mapping& mapping) {
+    os << "Mapping(evse: " << mapping.evse;
+    if (mapping.connector.has_value()) {
+        os << ", connector: " << mapping.connector.value();
+    }
+    os << ")";
+
+    return os;
+}
+
+/// \brief Writes the string representation of the given Mapping \p mapping to the given output stream \p os
+/// \returns an output stream with the Mapping written to
+inline std::ostream& operator<<(std::ostream& os, const std::optional<Mapping>& mapping) {
```

Comment:
The stream output operator should probably be defined for `Mapping` and not `std::optional<Mapping>`

Reply:
It already is: https://github.com/EVerest/everest-framework/blob/557932c30108ce09b3670f01cb49c5d5092b681f/include/utils/types.hpp#L81